pkg/minikube/node/node.go

```diff
@@ -122,13 +124,29 @@ func Delete(cc config.ClusterConfig, name string) (*config.Node, error) {
 		return n, err
 	}
 
+	cc.Nodes = append(cc.Nodes[:index], cc.Nodes[index+1:]...)
+	return n, config.SaveProfile(viper.GetString(config.ProfileName), &cc)
+}
+
+// Delete calls remove to remove node from cluster and deletes the host.
+func Delete(cc config.ClusterConfig, name string) (*config.Node, error) {
+	n, err := remove(cc, name)
+	if err != nil {
+		return n, err
+	}
+
+	m := config.MachineName(cc, *n)
+	api, err := machine.NewAPIClient()
+	if err != nil {
+		return n, err
```

Comment:
maybe i'm wrong, but in this place, it will return if it cannot get api client (needed for delete), not if it cannot load the config?

did you perhaps referred to the previous call to remove func? i do agree it should not return on err there, but continue and try to delete the host anyway, just not sure atm if we would also need to do anything additionally to clean up the leftovers